docs/source/documentation/4-docsearch-FAQ/2-not-an-open-source-project.html.md.erb

```diff
@@ -12,6 +12,6 @@ You are not an open source project? Feel free to ask, open an issue on our repo
 
 By focusing on open source projects, we want to make our contribution to the open source community. We are learning every day thanks to this incredible way of learning. we want to help. We do ask a link to the project repo, e.g. gitlab, github or any other terms of use that outlines this peculiarity.
 
-## My project is already featured or uses a back up documentation generator tool?
+## My project is already featured or uses a *baked* up documentation-generator tool?
```

Comment:
backed?

Reply:
We discussed this title. I didn't understand it. Feel free to help us. cc @s-pace

Reply:
TBH I don't really understand what the title / content is supposed to mean

Reply:
Yep thank you for sharing your concern @Haroenv 

I want to let people know that we will continue to provide DocSearch for website already using it and for new projects that use featured/supported documentation tools such as Docusaurus, gitbook, pkgdown, vuepress....

Reply:
>*baked in*

that would be a better choice then IMO